diff --git a/fnalibs/fnalibs.csproj b/fnalibs/fnalibs.csproj
index a9f2197..ef9c63f 100644
--- a/fnalibs/fnalibs.csproj
+++ b/fnalibs/fnalibs.csproj
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net4.0</TargetFramework>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <NoBuild>true</NoBuild>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>    
